File: C/basics/prime.c
#include <stdio.h>
int main()
{
    int num;
    int isprime = 1;
    printf("enter a prime number:");
    scanf("%d",&num);

    if (num <= 1)
    {
        isprime = 0;
    }
    else
    {
        int divisor = num / 2;
        while (divisor > 1)
        {
            if (num % divisor == 0)
            {
                isprime = 0;
                break;
            }
            divisor--;
        }
    }
    if (isprime)
    {
        printf("you enter the number is prime");
        
    }else{
        printf("you enter the number is not prime");
    }
    
    return 0;
}

File: javascript/prime.js
function isprime(n){
    if (n<2) {
             return false
    }
    for (let i = 2; i < n; i++) {
        if (n % 2 === 0){
            return false
        }
    
        
    }
    return true
}
const num = parseInt(prompt("Enter a number:"));

if (isprime(num)) {
  console.log(`${num} is a prime number.`);
} else {
  console.log(`${num} is not a prime number.`);
}
